runtime/src/main/scala/akka/grpc/internal/TelemetrySpi.scala

```diff
@@ -52,7 +52,7 @@ private[internal] object TelemetrySpi {
 @InternalStableApi
 trait TelemetrySpi {
   @nowarn
-  def onRequest(prefix: String, method: String, request: HttpRequest): Unit = {}
+  def onRequest[T <: HttpRequest](prefix: String, method: String, request: T): T = request
```

Comment:
Why was the type bound needed, something on the "other side"?

Reply:
yeah, so we can keep using the scala HttpRequest as a scala HttpRequest after going through the SPI